tests/test_runner.py

```diff
@@ -69,3 +71,35 @@ def test_add_first_appeared_dates():
     }
 
     assert runner.add_first_appeared_dates(probes_by_channel, first_appeared_dates) == expected
+
+
+def test_trailing_space():
+    '''Test cases to check the output of json.dumps has no trailing spaces
+    '''
+    test_cases = [
+        {
+            "test1": 12,
+            "test2": 31,
+        },
+        {
+            "test1": 12,
+            "test2": 31
+        },
```

Comment:
Both of these test cases are the same.  I think it's fine to just have one test case.

Reply:
yeah I will remove this portion